backend/docs/API_Specifications/facility-v2.yaml

```
@@ -0,0 +1,382 @@
+                items:
+                  $ref: '#/components/schemas/Facility'
+        '400':
+          description: Invalid query parameters
+  
+  /v2/facility/summary:
+    get:
+      tags:
+        - Facilities
+      summary: Get the health facility summary data
+      operationId: getFacilitiesSummary
+      parameters: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)